core/src/consensus/tendermint/worker.rs

```diff
@@ -709,20 +709,33 @@ impl Worker {
                 let parent_block_hash = self.prev_block_hash();
                 if let Some(priority_info) = self.signer_priority_info(parent_block_hash) {
                     if let TwoThirdsMajority::Lock(lock_view, locked_block_hash) = self.last_two_thirds_majority {
-                        cinfo!(ENGINE, "I am eligible to be a proposer, I'll re-propose a locked block");
+                        cinfo!(ENGINE, "I am eligible to be a proposer, round-info {}-{} and {}. I'll re-propose a locked block with priority",
+                            self.height,
+                            self.view,
+                            priority_info,
+                        );
```

Comment:
@HoOngEe Will you keep these logs?
Maybe we need a new log target like ENGINE_STATIC.

Reply:
Sorry, the messages were there before, I'll resolve this comment.